dev/automated_tests/integration_test/exception_handling_expectation.txt

```diff
@@ -33,9 +35,11 @@ When the exception was thrown, this was the stack:
 #2      main.<anonymous closure> \(.+[/\\]dev[/\\]automated_tests[/\\]integration_test[/\\]exception_handling_test\.dart:16:5\)
 <<skip until matching line>>
 The test description was:
-Exception handling in test harness - uncaught Future error
-<<skip until matching line>>
-  Test failed\. See exception logs above\.
+  Exception handling in test harness - uncaught Future error
+════════════════════════════════════════════════════════════════════════════════════════════════════
+\d\d:\d\d \+0 -3: Exception handling in test harness - uncaught Future error \[E\]
+  Test failed. See exception logs above.
   The test description was: Exception handling in test harness - uncaught Future error
+.*(TODO\(jiahaog\): Remove the next line once https://github.com/flutter/flutter/issues/81521 is fixed)?
```

Comment:
I'm correctly understanding this as effectively a comment in this file, not actually matching output right/

Reply:
That's right!